internal/env/env.go

```diff
@@ -0,0 +1,15 @@
+package env
+
```

Comment:
If you need a separate package you can use `confutil` call this `exp`

Reply:
Moved.